### DISCUSSION TITLE
Can I use keras_cv with GPU on windows?

### DISCUSSION THREAD

**Discussion by Man-Heim** [keras-team/keras-cv#2398] [Q&A]
I am using keras_cv to create an object detection with yolov8. I've built the model with pretrained weights and a pretrained backbone directly from keras_cv. I also use methods like resizing from keras_cv. Overall with my current implementation I heavily rely on keras_cv inspired by this tutorial: https://keras.io/examples/vision/yolov8/#introduction

Up until now I've only created the model and a data pipeline, logging and so on with the use of my CPU and now I want to start real trainings on my GPU. Since Tensorflow with GPUs works only on Windows when I use Tensorflow version 2.10 ("Caution: TensorFlow 2.10 was the last TensorFlow release that supported GPU on native-Windows." from Tensorflow website), I've downgraded to Tensorflow 2.10. Now I have the problem that keras_cv is not compatible with Tensorflow Versions lower than 2.11.

So my question now is: Is it not possible to use Keras_cv on Windows with GPU support. Because I can't find a way how to get GPU support for windows working with a Tensorflow version higher than 2.10. but I also can't find a way how keras can run with Tensorflow versions  lower than 2.11.

**Comment by innat** [comment]
> Because I can't find a way how to get GPU support for windows working with a Tensorflow version higher than 2.10. 

Use WSL. It's worth it.
